R/getGeneticInteractome.R

```diff
@@ -0,0 +1,42 @@
+# getGeneticInteractome.R
+#' @title getGeneticInteractome
+#' @author Nada Elnour, \email{nada.elnour@@mail.utoronto.ca}
+#'
+#' \code{getGeneticInteractome} augments systems with BioGRID genetic
+#' interaction annotations
+#'
+#' @param mySys The dataframe of PPI between system's components
+#' @param criterion A string, either "stringent" or "relaxed", specifying
+#' whether only GGI between physical interactors should be selected.
+#'
+#' @return The dataframe of system components and either their GGI between
+#' physical interactors should be selected (iff \code{criterion} == "stringent")
+#' or all GGI of physical interactors (iff \code{relaxed} == "stringent")
+#'
```

Comment:
@example + short explanation is needed for every exported function

Reply:
fixed